### ISSUE TITLE
[FEATURE] RMA index

### ISSUE THREAD

**Issue by mastertnt** [ho-dev/HattrickOrganizer#2117]
**Is your feature request related to a problem? Please describe.**
**As** a  user
**I want to** have the RMA (rate-my-academy) skill index in the youth tab
**in order to** have a comprehensive value to select my best young.

**Describe the solution you'd like**
Add a new column into the overview table

**Additional context**
Keep the same color for cells as "Rate my academy"

**Comment by mastertnt** [comment]
If can try to develop it :)

**Comment by wsbrenk** [comment]
you're welcome to help improving the HO. How do you want to get these numbers. Download from https://www.rate-my.academy/players or do you know the formula?

**Comment by mastertnt** [comment]
The formula is available

**Comment by mastertnt** [comment]
Ok so the process is to rease a PR ?

**Comment by wsbrenk** [comment]
PR is the correct way.

Concerning translation, you have to introduce the property in  src/main/resources/sprache/english.properties (or ...language/english.properties, when @sgcr  pull request is merged).

Currently the integration with poeditor is a little bit broken  - so the property file has to be imported to poeditor by hand. And there is currently another issure #2116. So i'm really thinking about removing the HO poeditor project and maintain the properties again in our repo.

**Comment by mastertnt** [comment]
Ok thanks, i will do it. 

Last question, before PR, the skills are updated accordingly to a strategy, so the real values from HO are lost (e.g max keeper level). Does it make sense to lost those values ? I understand the strategy but in my point of view, those values must be computed from DB sources not injected as "real" values.

**Comment by wsbrenk** [comment]
Currently the skill levels (numbers without precisions) for start, current and maximum are downloaded values from HT and should not be changed.

Current and start skill values can be adjusted by the user within the ranges allowed by the downloaded ranges. Overall skill rule is also respected.

If your algorithm will set these values initially, it would be a great help. User changes should be possible to overwrite your settings.

**Comment by mastertnt** [comment]
No, what i said the values are updated by the previous approach (in the methods :  setPlayerMaxSkills and areKeeperSkills. If the player is considrer as goalkeeper, the max values are changed. It seems curious to me.

**Comment by wsbrenk** [comment]
~~~
    /**
     * If player is a GoalKeeper, maximum skills of winger, playmaking, passing and scoring are set (4)
     * otherwise the maximum of the keeper skill is set to 4 (field player)
     *
     * @param isKeeper true, keeper maximums are set
     *                 false, field player maximum is set
     */
    public void setPlayerMaxSkills(boolean isKeeper)
 ~~~
 
 Ah, you're talking about this!? But this is the rule in hattrick as far as I know.

**Comment by wsbrenk** [comment]
Now i see your point - setting the maximum to the value of 4 is of course not the real maximum, which could be lower than 4. If this real limit will be mentioned by the trainer, this lower value should be set (Hopefully). 
If you want, you can implement another attribute for this "preliminary" maximum value (which does not need to be stored, but recalculated each time the player values are loaded. 
in the player overview those limited skills should be tagged as bad (red) skills.

**Comment by mastertnt** [comment]
Ok il will do it like that, thx.

**Comment by mastertnt** [comment]
> ```
>     /**
>      * If player is a GoalKeeper, maximum skills of winger, playmaking, passing and scoring are set (4)
>      * otherwise the maximum of the keeper skill is set to 4 (field player)
>      *
>      * @param isKeeper true, keeper maximums are set
>      *                 false, field player maximum is set
>      */
>     public void setPlayerMaxSkills(boolean isKeeper)
> ```
> 
> Ah, you're talking about this!? But this is the rule in hattrick as far as I know.

Yes

**Comment by wsbrenk** [comment]
Found it here: https://www86.hattrick.org/Forum/Read.aspx?t=17379585&n=1&v=4&mr=0

Current scoring logic:

*  For all known potentials the following formula is used: x*0.87^(8-(1...y)) where x is a fixed value based on the skill training speed (see table below) and y is the player's potential in the skill in question. For example for a solid potential in playmaking the score would be (200*0.87^(8-1))+(200*0.87^(8-2))+...+(200*0.87^(8-7))≈834.

* If only current skill is known (x/?) current skill + 1 is used.

* For unknown skills overall skill - 1 is used, or if overall skill is unknown then default value 3 is used (GK is always 1 and SP is always 3 regardless of overall skill).

* Sum of relevant potentials are used to get total score. For field players this means all but goalkeeping and for goalkeepers formula (gk+def+sp)*2.2 is used. Whichever sum is greater is used as the final score.

* Specialty awards points with the following logic:
  * Field players get +200 points for all specialties except for resilient which is worth +100 points
  * Goalkeepers get +100 points for all specialties

* Players who can't be promoted at 17.00 suffer a 10p/exceeding day penalty.


Weighted values for skills
Gk	130
Def	230
Pm	200
Wg	140
Ps	180
Sc	210
Sp	40
